app/Http/Controllers/ContactUsController.php

```diff
@@ -1,12 +1,39 @@
 <?php
-
 namespace App\Http\Controllers;
-
-use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use App\Http\Requests\ContactUsRequest;
+use Illuminate\Mail\Message;
 
 class ContactUsController extends Controller
 {
-    public function index() {
-        return view('contact-us');
+    public function index()
+    {
+        return view('contacts.contactUs');
+    }
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        $callback = function (string $test): string{
+            return "{$test} input";
+        };
+
+        $data = $request->validated();
+        \Log::debug($callback('test'), $data);
+        \Mail::send(
+            'emails.contactUs', 
+            [
+                'email' => $data['email'],
+                'name' => $data['name'],
+                'phone' => $data['phone'],
+                'services' => $data['services'],
+                'messageText' => $data['message'],
+            ],
+            function(Message $message) use ($data){
+                $message->subject('Message from customer ' . $data['email']);
+                $message->to('tech@baloon.com');
```

Comment:
Please adjust to and from to use your project related values.

Reply:
Adjusted for my project